src/fhir.js

```diff
@@ -653,7 +654,16 @@ function toCode(f) {
     return f.value;
   }
 }
+function getClassInfo(profile, retrievedDetails, _modelInfo) {
+  let classInfo = [];
```

Comment:
```suggestion
  let classInfo = null;
```

Should be returning null, not `[]`, if we can't find anything